src/ex4/dist/scripts/Main.js

```diff
@@ -0,0 +1,304 @@
+import { ItemManager } from "../clients/ItemManager.js";
+import { Alert } from "./Alert.js";
+const EMPTY_INPUT_MSG = "Task input cannot be empty, please try again";
+
+class main {
+  constructor(htmlElement) {
+    this.tasksManager = new ItemManager();
+    this.emprtyInputAlert = new Alert(htmlElement);
+
+    // Action Elements //
+
+    this.taskInput = htmlElement.querySelector(".task-input");
```

Comment:
do we really need all those elements on "this"?
consider having only the ones you reuse in different functions here.
you can query the other elements only when you need them.